rootfs/ingest/python/qserv/ingest.py

```diff
@@ -312,9 +305,7 @@ def _get_chunk_location(repl_url, chunk, database, transaction_id):
                "transaction_id": transaction_id}
     responseJson = Http().post(url, payload)
 
-    # Get location host and port
-    host = responseJson["location"]["host"]
-    port = 25004
+    host, port = jsonparser.get_location(responseJson)
```

Comment:
I wonder why you separate code that sends REST requests (`Http().post(...)`) from the one that analyzes responses `jsonparser.*`. It would be more convenient (and safer) if you had a class encapsulating both operations, so that you could do things like:
```
host, port = ReplicationSystemAPI().get_location(url, options=...)
```
The you won't need to carry around temporary objects like `responseJson`.

Reply:
Yes this is a good idea. I was splitting them for unit testing purpose but this is not needed and I might merge them in the near term.